src/main/java/org/wiremock/RandomHelper.java

```diff
@@ -0,0 +1,31 @@
+package org.wiremock;
+
+import com.github.jknack.handlebars.Options;
+import com.github.tomakehurst.wiremock.extension.responsetemplating.helpers.HandlebarsHelper;
+import net.datafaker.Faker;
+
+/*
+ * Author: Shreya Agarwal
+ */
+public class RandomHelper extends HandlebarsHelper<Object> {
+
+  private final Faker faker;
```

Comment:
Is the implementation thread safe? I would recommend avoiding creating fields in Handlebars resolvers because IIRC they are cached

Reply:
Oh I see, let me check this once I'm off work.

Reply:
@oleg-nenashev I don't see anything in the docs regarding thread safety but my guess is that it is, given that the APIs exposed by it seem stateless.

> I would recommend avoiding creating fields in Handlebars resolvers because IIRC they are cached

Why is that? Helpers objects are created only once in the lifetime of service bootup, so it should be fine to have a field, right?

Reply:
As far as I can tell it's thread safe as it's a bunch of stateless functions.

The only downside of having a singleton Faker instance is it won't work if/when we want to do random seed pinning as the randomiser has to be passed to its constructor.

However, given how much work the constructor does I think it'd be worth it for now anyway.

Reply:
Makes sense, for the random seed pinning I was planning to create a new `Faker` instance with the seed, else default to using the existing instance, but that's a discussion for when we come to that feature.

What do you suggest I do now? Leave it as is or create a new instance everytime?

Reply:
I think your plan is a good one, so let's have a singleton Faker instance and we'll add per-call instances once the plumbing is in place to do seed pinning